src/components/util/ask-text-dialog.js

```diff
@@ -29,10 +29,19 @@ import PropTypes from 'prop-types';
 export const AskTextDialog = ({ title, value, show, onValidate, onClose }) => {
     const [currentValue, setCurrentValue] = useState(value);
     const intl = useIntl();
+
+    useEffect(() => {
```

Comment:
We need both useEffect here ?

Reply:
nope, just an auto merge with another, will remove the second one